DEVEL.md

```diff
@@ -207,11 +207,16 @@ Generate package manifests with:
 $ make packagemanifests OPERATOR_VERSION=0.X.Y
 ```
 
+Copy the generated files to the Community Operators repo:
+```
+$ cp -r packagemanifests/* community-operators/operators/intel-device-plugins-operator/
```

Comment:
where community-operators directory came from?

Reply:
Insted of copying files we can use -output-dir operator-sdk option to put files to the right place when generating them